### PR TITLE
readme cleanup

### PR DESCRIPTION
**Pull request by ecbrodie**
This is an awesome project! I thought that I could help by fixing up your `README.md` a little bit, if you don't mind:

* Corrects some English grammar
* Adds code formatting around filename in the first sentence
* Changes the wording about RuboCop versions so that a README.md edit does not need to occur every time RuboCop releases a new version

Feel free to drop any of my suggestions if you do not desire them, I won't mind.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Mry
 
-Mry Migrates .Rubocop.Yml :muscle:
+Mry Migrates `.rubocop.yml` :muscle:
```
Comment (pocke) [inline README.md]: Could you revert this change? The capital characters mean acronym.

Migrates
Rubocop
Yml

**Comment by ecbrodie** [comment]
Ah, I get it! Will do.

**Comment by ecbrodie** [comment]
@pocke please review again, thanks.

**Comment by pocke** [comment]
Thank you!
